src/encoded/static/components/app.js

```diff
@@ -945,13 +943,6 @@ class App extends React.Component {
         return true;
     }
 
```

Comment:
We still need the `handleBeforeUnload` method here. The part we didn’t need was the call to `cartIsUnsaved`. Without `handleBeforeUnload`, if you reload the page after changing an object with the form, we don't get an alert letting the user back out of reloading.